docs/providers/snowflake/index.rst

```diff
@@ -6,6 +6,8 @@ Snowflake Example DAG
   :maxdepth: 2
 
     Snowflake Operators <operators/snowflake>
+    Snowflake Sql Api Operators <operators/snowflake>
```

Comment:
```suggestion
    Snowflake Sql Api Operators <operators/snowflake_sql_api>
```

Reply:
Also, if you want you can run `make docs` then open `docs/_build/html/index.html` in your local browser and check how it is looking